### DISCUSSION TITLE
Auto update

### DISCUSSION THREAD

**Discussion by vdvmichel** [TypQxQ/Sigenergy-Home-Assistant-Integration#28] [Ideas]
Is it possible to make a hacs integration? Then we can do auto update when you change something…

**Comment by TypQxQ** [comment]
This requires a complete rewrite to Python and is not something I prioritize.
It could add more features like DC and AC charging addons with a click, multiple SigenStor connected to a single Sigen ESS.
But maybe Sigenergy will develop their own official plugin, who knows...

**Comment by vdvmichel** [comment]
I personally think that converting to a hacs integration will result in more users and more contributors, making the integration even better...
